repository/repo.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose, { Collection } from 'mongoose';
+
+mongoose.set('strictQuery', false);
+const mongoDB = "mongodb://127.0.0.1:27017/test"; 
```

Comment:
All configuration parameters (such as urls) should be written in a `.env` file.

You should use the dotenv module to use them